src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public abstract double getArea();
+    public abstract void draw();
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the beginning of the `Figure` class definition (line 3) and method definitions (lines 6 and 13).